gui/ThemeEngine.cpp

```diff
@@ -1568,20 +1565,26 @@ bool ThemeEngine::createCursor(const Common::String &filename, int hotspotX, int
 	_cursorWidth = cursor->w;
 	_cursorHeight = cursor->h;
 
-#ifdef USE_RGB_COLOR
-	_cursorFormat = cursor->format;
-	_cursorTransparent = _cursorFormat.RGBToColor(0xFF, 0, 0xFF);
+	_cursorTransparent = 255;
+	_cursorFormat = Graphics::PixelFormat::createFormatCLUT8();
```

Comment:
Hmmm, we load cursors from SVGs or BMPs, both are truecolor. How are we now forcing it to CLUt8?

Reply:
Cursor graphics are handled like this:
- If the backend supports alpha transparency for cursors, the graphic is used as-is.
- If the backend doesn't support alpha transparency but does support cursor palettes, the graphic is reduced to 256 colours before being passed to the backend.
- If the backend supports neither of these features (which shouldn't happen with any of the current backends), the graphic is ignored and a default crosshair cursor is used instead.